torchvision/models/detection/transform.py

```diff
@@ -56,6 +62,17 @@ def normalize(self, image):
         return (image - mean[:, None, None]) / std[:, None, None]
 
     def resize(self, image, target):
+        if self.is_xla:
+            # Resize to a fixed size on TPU to avoid dynamicity.
+            new_h, new_w = 600, 800
+            image = torch.nn.functional.interpolate(
+                image[None], size=(new_h, new_w), mode='bilinear', align_corners=False)[0]
+
+            if target is None:
+                return image, target
+ 
+            raise NotImplementedError
```

Comment:
can you explain why we only return if target is None here?

Reply:
If you scroll down you can see it's the original behavior of that method. Basically, in inference mode (as opposed to training), `target` is None and therefore there's no operations to run on it. I haven't implemented resizing of the boxes/masks/keypoints for training mode yet, so that's why I throw an exception.